crates/rpc/rpc-eth-api/src/helpers/fee.rs

```diff
@@ -44,6 +45,48 @@ pub trait EthFees: LoadFee {
         LoadFee::suggested_priority_fee(self)
     }
 
+    fn pending_block_to_fee_entry(
+        &self,
+        pending_block: &SealedBlockWithSenders,
+        pending_receipts: &[Receipt],
+    ) -> Result<FeeHistoryEntry, Self::Error> {
```

Comment:
hmm, we kinda have this already:

https://github.com/paradigmxyz/reth/blob/6ba4bbe4aad49e483b0670903bdb0328d9ca1b57/crates/rpc/rpc-eth-types/src/fee_history.rs#L85-L93

would be nice to reuse this, this could even be a function of FeeHistory itself

Reply:
ohh, I didn't see it